docs/config/README.md

```diff
@@ -120,11 +120,12 @@ The `aws` module shows the current AWS region and profile. This is based on
 
 ### Options
 
-| Variable   | Default         | Description                                                |
-| ---------- | --------------- | ---------------------------------------------------------- |
-| `symbol`   | `"☁️  "`        | The symbol used before displaying the current AWS profile. |
-| `style`    | `"bold yellow"` | The style for the module.                                  |
-| `disabled` | `false`         | Disables the `AWS` module.                                 |
+| Variable          | Default         | Description                                                |
+| ----------        | --------------- | ---------------------------------------------------------- |
+| `symbol`          | `"☁️  "`         | The symbol used before displaying the current AWS profile. |
+| `style`           | `"bold yellow"` | The style for the module.                                  |
+| `disabled`        | `false`         | Disables the `AWS` module.                                 |
+| `displayed_items` | `All`           | Chose which item to display. Possible values [`All`, `Profile`, `Region`] |
```

Comment:
For consistency with other string configuration options, like `style`, I think it would be best if the values were lowercase.
```suggestion
| `displayed_items` | `All`           | Chose which item to display. Possible values [`all`, `profile`, `region`] |
```

Reply:
Done, it might be possible to convert the value to lowercase at run time to make it case insensitive. For now, I'll just make it lowercase.